### ISSUE TITLE
How to get output from last layer 

### ISSUE THREAD

**Issue by jaja136** [jaja136/AI-and-Deep-Learning#1]
model = Sequential()
input_layer = model.add(Dense(64, input_dim=40, activation='relu'))
model.add(Dropout(0.5))
hidden_layer1 = model.add(Dense(64, activation='relu'))
model.add(Dropout(0.5))
hidden_layer2 = model.add(Dense(64, activation='relu'))
model.add(Dropout(0.5))
hidden_layer3 = model.add(Dense(64, activation='relu'))
model.add(Dropout(0.5))
hidden_layer4 = model.add(Dense(64, activation='relu'))
model.add(Dropout(0.5))
hidden_layer5 = model.add(Dense(64, activation='relu'))
model.add(Dropout(0.5))
output_layer = model.add(Dense(1, activation='sigmoid'))

model.compile(loss='binary_crossentropy',  optimizer='rmsprop',  metrics=['accuracy'])
